src/game.cpp

```diff
@@ -4407,6 +4407,43 @@ void Game::addDistanceEffect(const SpectatorVec& spectators, const Position& fro
 	}
 }
 
+void Game::setAccountStorageValue(const uint32_t accountId, const uint32_t key, const int32_t value)
+{
+	if (value == -1) {
+		accountStorageMap[accountId].erase(key);
+		return;
+	}
+
+	accountStorageMap[accountId][key] = value;
+}
+
+bool Game::getAccountStorageValue(const uint32_t accountId, const uint32_t key, int32_t& value) const
```

Comment:
Return a `pair<bool, int32_t>` or just return `-1` as it was defined to be the invalid value